From b49ef9b81d78874530e746d431273869a59876ee Mon Sep 17 00:00:00 2001
From: Charles Munson <jetwhiz@jetwhiz.com>
Date: Tue, 24 May 2016 13:24:19 +0200
Subject: Do not send logs to output file (easyloggingpp)


diff --git a/src/easylogging++.h b/src/easylogging++.h
index f9a67fe..c392c62 100755
--- a/src/easylogging++.h
+++ b/src/easylogging++.h
@@ -789,7 +789,7 @@ namespace consts {
     static const char* kDefaultLogFile                         =      "logs/myeasylog.log";
 #      endif  // ELPP_OS_ANDROID
 #   elif ELPP_OS_WINDOWS
-    static const char* kDefaultLogFile                         =      "logs\\myeasylog.log";
+    static const char* kDefaultLogFile                         =      "NUL";
 #   endif  // ELPP_OS_UNIX
 #endif  // defined(ELPP_DEFAULT_LOG_FILE)
 #if !defined(ELPP_DISABLE_LOG_FILE_FROM_ARG)
